src/ui_interface.cpp

```diff
@@ -54,13 +54,22 @@ void CClientUIInterface::NotifyBlockTip(bool b, const CBlockIndex* i) { return g
 void CClientUIInterface::NotifyHeaderTip(bool b, const CBlockIndex* i) { return g_ui_signals.NotifyHeaderTip(b, i); }
 void CClientUIInterface::BannedListChanged() { return g_ui_signals.BannedListChanged(); }
 
-
 bool InitError(const std::string& str)
 {
     uiInterface.ThreadSafeMessageBox(bilingual_str{str, str}, "", CClientUIInterface::MSG_ERROR);
     return false;
 }
 
+bool InitError(const bilingual_str& str)
+{
+    bilingual_str bs{str.original, str.translated};
```

Comment:
a9641a7ac12e3eae763faadbf9bfeeb5f14e3717

Just
```cpp
bilingual_str bs{str};
```

Reply:
Agree. Will be fixed tonight.

Reply:
Done.

Reply:
It fails on trusty (https://travis-ci.org/bitcoin/bitcoin/jobs/554836240).

Reply:
iono about that error 😕 somehow it doesn't use copy constructor.

Reply:
[Copy initialization](https://en.cppreference.com/w/cpp/language/copy_initialization) will do:
```suggestion
    bilingual_str bs = str;
```